# This file is automatically generated by Posit Publisher; do not edit.
'$schema' = 'https://cdn.posit.co/publisher/schemas/posit-publishing-record-schema-v3.json'
server_type = 'connect_cloud'
server_url = 'https://connect.posit.cloud'
client_version = '1.22.0'
created_at = ''
dismissed_at = ''
type = 'python-shiny'
configuration_name = 'csrbsr_test1-LQ7L'
id = '01998764-15e9-3d35-e3d4-21e5092235d4'
dashboard_url = 'https://connect.posit.cloud/sfanghdr/content/01998764-15e9-3d35-e3d4-21e5092235d4'
direct_url = 'https://01998764-15e9-3d35-e3d4-21e5092235d4.share.connect.posit.cloud'
logs_url = 'https://connect.posit.cloud/sfanghdr/content/01998764-15e9-3d35-e3d4-21e5092235d4/history'
deployed_at = '2025-09-26T14:58:52-04:00'
bundle_id = '01998764-15ea-497b-9c7d-d552fc795daa'
files = [
  '.posit/publish/csrbsr_test1-LQ7L.toml',
  '.posit/publish/deployments/deployment-2064.toml',
  'app.py',
  'requirements.txt'
]
requirements = [
  "ipywidgets==7.8.1 # ipyleaflet's popup is broken with >8.0",
  'ipyleaflet',
  'shiny @ git+https://github.com/posit-dev/py-shiny@main',
  'shinywidgets',
  'plotly',
  'matplotlib',
  'scipy',
  'pandas',
  'faicons',
  'streamstats',
  'geopandas',
  'matplotlib',
  'pyproj',
  'python-math',
  'ezdxf'
]

[connect_cloud]
account_name = 'sfanghdr'

[deployment_error]
code = 'unknown'
message = 'Runtime error: Unable to start the app due to a runtime error. Please check the logs for more information.'
operation = 'publish/deployContent'

[configuration]
product_type = 'connect_cloud'
'$schema' = 'https://cdn.posit.co/publisher/schemas/posit-publishing-schema-v3.json'
type = 'python-shiny'
entrypoint = 'app.py'
validate = true
files = [
  '/app.py',
  '/requirements.txt',
  '/.posit/publish/csrbsr_test1-LQ7L.toml',
  '/.posit/publish/deployments/deployment-2064.toml'
]
title = 'csrbsr_test1'

[configuration.python]
version = '3.12'
